lib/fixer/configuration.rb

```diff
@@ -20,6 +22,9 @@ module Configuration
     # The value sent in the http header for 'User-Agent' if none is set
     DEFAULT_USER_AGENT = "Fixer Ruby Gem #{Fixer::VERSION}".freeze
 
+    # sqs queue to write messages
+    DEFAULT_QUEUE = 'production_fixer_job_create'.freeze
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.